crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -19,103 +19,79 @@ pub use {auction::Auction, solution::Solution, transaction::Transaction};
 #[allow(dead_code)]
 #[derive(Debug)]
 pub struct Settlement {
-    solution: Solution,
-    transaction: Transaction,
+    settled: Transaction,
```

Comment:
Don't forget to update the docs (it's no longer a solution together with an auction). Please be extra careful with making sure these structs have documentation that give the reader context on what their scope is and how they differ from similarly named structs (not just a verbatim copy of the field names). `setttled::Transaction` is a good example 👍

Reply:
updated comments 👍